lib/auth/middleware_test.go

```diff
@@ -789,6 +789,8 @@ func TestRejectedClientClusterAlert(t *testing.T) {
 	require.Len(t, alerts, 1)
 	require.Equal(t, "rejected-unsupported-connection", alerts[0].GetName())
 	require.Contains(t, alerts[0].Spec.Message, "agents")
+	// Assert that the version in the message does not contain any prereleases
+	require.NotContains(t, alerts[0].Spec.Message, "-aa")
```

Comment:
Are there any test cases that inject a prerelease version so that this test will assert that this filter is working?